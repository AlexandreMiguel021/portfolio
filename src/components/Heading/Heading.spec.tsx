import { render } from '@testing-library/react'
import { Heading } from '.'
import '@testing-library/jest-dom'

describe('Heading', () => {
	it('should render a Heading correctly', () => {
		const { container } = render(<Heading>Content</Heading>)
		expect(container).toMatchSnapshot()
	})
	it('should render a Heading with children', () => {
		const { getByText } = render(<Heading>Content</Heading>)
		expect(getByText('Content')).toBeInTheDocument()
	})
})
