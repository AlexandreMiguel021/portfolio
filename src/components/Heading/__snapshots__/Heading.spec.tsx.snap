// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render a Heading correctly 1`] = `
<div>
  <h2
    class="heading"
    data-aos="fade-up"
  >
    Content
  </h2>
</div>
`;
